From private repo

v-alje opened the pull request.

azurecla commented:
Hi __@deneha__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>You've already signed the contribution license agreement. Thanks!</span>
        <span>We will now validate the agreement and then real humans will evaluate your PR.</span>

TTYL, AZPRBOT;

carolinacmoravia committed:
M14068: Incorrect word: "has"

Hello, @chackdan,

Translator has reported possible source content issue.  It seems that "has" should be "hash" since "has" does not make sense in this context.
Please, help to check my comment tag added into the article and reply with explanation if source text should be fixed, or it is by design. If fix is needed, please, let me know either if you would like me to fix it in this PR, or if you will fix it in another PR. In case of using another PR, please, let me know of your PR number, so we can confirm and close this PR. 
Many thanks in advance.

gsacavdm committed:
Add link to connector releases page.

bwren committed:
Azure monitoring

bwren committed:
updates

bwren committed:
updates

iainfoulds committed:
Updating info on sizes per PM feedback

iainfoulds committed:
Acrolinx updates

parakhj committed:
Updating billing doc

bwren committed:
video

parakhj committed:
Updating links

JasonWHowell committed:
Updating broken AZ CLI links

parakhj committed:
Fixing feedback

parakhj committed:
Fixing casing

spelluru committed:
compare v1 and v2

bwren committed:
remove monitoring article

SnehaGunda committed:
Fixing policy module

tdykstra committed:
add don't mix local and portal dev note

bwren committed:
test without video

rwike77 committed:
updated links

v-shils committed:
Merge pull request #28918 from parakhj/master

Updating billing doc

tfosmark committed:
Merge pull request #28516 from gsacavdm/patch-1

Add link to connector releases page.

rwike77 committed:
updated toc

rwike77 committed:
updated text

bwren committed:
video

JasonWHowell committed:
Update app-service-api-create-resource-group.md

Ja-Dunn committed:
Merge pull request #28912 from iainfoulds/vmlinuxoverview

VM Linux overview updates

Commit:
Merge pull request #28978 from tdykstra/patch-11

add don't mix local and portal dev note

PRMerger18 committed:
Merge pull request #28972 from bwren/la-changes

Log Analytics changes video

Ja-Dunn committed:
Merge pull request #28939 from JasonWHowell/includebrokenlinks

Updating broken AZ CLI links

PRMerger14 committed:
Merge pull request #28980 from SnehaGunda/mentions

Fixing policy module

Ja-Dunn committed:
Merge pull request #28964 from spelluru/comparev1v21130

compare v1 and v2

Ja-Dunn committed:
Merge pull request #28989 from rwike77/cluster-tutorial

updated links

PRMerger13 committed:
Merge pull request #28000 from carolinacmoravia/patch-462

M14068: Incorrect word: "has"

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs into FromPublicRepo

v-alje committed:
Merge pull request #29008 from MicrosoftDocs/FromPublicRepo

From public repo
